src/main/java/com/cloudogu/wiki/RefreshWikisServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.cloudogu.wiki;
+
+import com.cloudogu.wiki.scmm.SessionCacheScmWikiListStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
```

Comment:
Please add a short description of what the servlet does. Something like removes the cached wiki list from session.